### PR TITLE
Update the instructions for querying bad data

### PR DESCRIPTION
**Pull request by stanch**

### CODE REVIEW

File: docs/managing-data-quality/failed-events/failed-events-in-athena-and-bigquery/index.md [hunk 1]
```diff
@@ -116,6 +95,51 @@ bq mk \
...
+
+```sql
+SELECT message.field AS field,
+       message.value AS value,
+       message.error AS error,
+       message.json AS json,
+       message.schemaKey AS schemaKey,
+       message.schemaCriterion AS schemaCriterion
+FROM schema_violations
+CROSS JOIN UNNEST(data.failure.messages) AS t(message)
```
Comment (stanch) [inline docs/managing-data-quality/failed-events/failed-events-in-athena-and-bigquery/index.md:135]: Added the unnest like we have in the repo

File: docs/managing-data-quality/failed-events/failed-events-in-athena-and-bigquery/index.md [hunk 2]
```diff
@@ -146,22 +169,28 @@ GROUP BY data.payload.enriched.app_id;
...
-       data.failure.messages[OFFSET(0)].schemaKey AS schemaKey,
-       data.failure.messages[OFFSET(0)].schemaCriterion AS schemaCriterion
-FROM bad_rows_prod1.tracker_protocol_violations
+SELECT message.field AS field,
+       message.value AS value,
+       message.expectation AS expectation,
+       message.schemaKey AS schemaKey,
+       message.schemaCriterion AS schemaCriterion
+FROM bad_rows_prod1.tracker_protocol_violations,
+unnest(data.failure.messages) AS message
```
Comment (stanch) [inline docs/managing-data-quality/failed-events/failed-events-in-athena-and-bigquery/index.md:178]: Also added unnest

File: docs/managing-data-quality/failed-events/failed-events-in-athena-and-bigquery/index.md [hunk 3]
```diff
@@ -4,81 +4,54 @@ date: "2020-04-30"
...
-ORDER BY data.failure.timestamp DESC
-LIMIT 10
-```
-
-## BigQuery instructions
+:::note
 
 These instructions make use of the [bq command-line tool](https://cloud.google.com/bigquery/docs/bq-command-line-tool) which is packaged with the [google cloud sdk](https://cloud.google.com/sdk/docs). Follow the sdk instructions for how to [initialize and authenticate the sdk](https://cloud.google.com/sdk/docs/initializing). Also take a look at the [BigQuery dashboard](https://console.cloud.google.com/bigquery) as you run these commands, so you can see your tables as you create them.
 
-##### Missing fields ⚠️
```
Comment (stanch) [inline docs/managing-data-quality/failed-events/failed-events-in-athena-and-bigquery/index.md:71]: Removing thanks to https://github.com/snowplow-incubator/snowplow-badrows-tables/pull/14

**Review by mharvey83 — APPROVED** [review]
LGTM. Some really nice improvements to simplify the docs

File: docs/managing-data-quality/failed-events/failed-events-in-athena-and-bigquery/index.md [hunk 4]
```diff
@@ -146,22 +180,91 @@ GROUP BY data.payload.enriched.app_id;
...
+  ]
+}
+```
+
+You can figure out what to expect from such a field by looking at the JSON schema for the respective type of failed events, in this case the [tracker protocol violations schema](https://github.com/snowplow/iglu-central/blob/master/schemas/com.snowplowanalytics.snowplow.badrows/tracker_protocol_violations/jsonschema/1-0-0).
+
+BigQuery has a variety of JSON functions that allow you to extract data from within complex objects. For instance, if you are interested in Iglu repositories that failed to resolve, you can use something like this:
+
+```sql
+SELECT DISTINCT(JSON_VALUE(message.error.lookupHistory[0].repository))
```
Comment (stanch) [inline docs/managing-data-quality/failed-events/failed-events-in-athena-and-bigquery/index.md:235]: I’ve tested this one and the one below

**Review by oguzhanunlu — APPROVED** [review]
💯 thanks Nick!

**Pull request merged**